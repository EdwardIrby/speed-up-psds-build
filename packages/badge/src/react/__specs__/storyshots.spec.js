import path from 'path'
import initStoryshots from '@storybook/addon-storyshots'

jest.mock('@pluralsight/ps-design-system-storybook-addon-center')

initStoryshots({
  configPath: path.resolve(__dirname, '..', '..', '..', '.storybook')
})
