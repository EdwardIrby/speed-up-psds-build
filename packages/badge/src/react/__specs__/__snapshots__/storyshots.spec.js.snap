// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots appearance default 1`] = `
<div
  data-css-1so76ss=""
>
  <div
    style={
      Object {
        "display": "grid",
        "gridGap": 10,
      }
    }
  >
    <div>
      <div
        data-css-1la7nrz=""
      >
        default
         
        gray
      </div>
    </div>
    <div>
      <div
        data-css-157u8fk=""
      >
        default
         
        green
      </div>
    </div>
    <div>
      <div
        data-css-o5sj1x=""
      >
        default
         
        yellow
      </div>
    </div>
    <div>
      <div
        data-css-oh7fxh=""
      >
        default
         
        red
      </div>
    </div>
    <div>
      <div
        data-css-18fz1k5=""
      >
        default
         
        blue
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots appearance subtle 1`] = `
<div
  data-css-1so76ss=""
>
  <div
    style={
      Object {
        "display": "grid",
        "gridGap": 10,
      }
    }
  >
    <div>
      <div
        data-css-ikerdr=""
      >
        subtle
         
        gray
      </div>
    </div>
    <div>
      <div
        data-css-hmqqyn=""
      >
        subtle
         
        green
      </div>
    </div>
    <div>
      <div
        data-css-inha5s=""
      >
        subtle
         
        yellow
      </div>
    </div>
    <div>
      <div
        data-css-1txlinh=""
      >
        subtle
         
        red
      </div>
    </div>
    <div>
      <div
        data-css-7j87uz=""
      >
        subtle
         
        blue
      </div>
    </div>
  </div>
</div>
`;
